string arm_name          # left_arm / right_arm / both_arms
string command_type      # gripper
string gripper_state     # open / close